.circleci/config.yml

```diff
@@ -169,13 +297,42 @@ workflows:
           filters:
             branches:
               ignore: main
+      - bump-dev-version:
+          filters:
+            branches:
+              only:
+                - develop
+                - /.*-publish-dev-package/
+          requires:
+            - pip-install
+      - build-package:
+          name: build-dev-package
+          requires:
+            - webpack
+            - compile-translations
+            - bump-dev-version
+      - publish-package:
+          name: publish-dev-package
+          context: pypi-test
+          filters:
+            branches:
+              only:
+                - develop
+                - /.*-publish-dev-package/
+          requires:
+            - build-dev-package
+
   main:
     jobs:
       - pip-install:
           name: pip-install-main
           filters:
             branches:
               only: main
+      - bump-version:
+          context: deliverino
+          requires:
+            - pip-install-main
```

Comment:
```suggestion
```
You cannot reference a job that doesn't exist yet. You only defined `bump-dev-version`, not `bump-version`.
Also, this PR is only about the develop workflow #11, so you don't need to add jobs for the deploy workflow. The deployment workflow is covered by #36.